projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobDeployment.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.service.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Entity
+public class DataJobDeployment {
+
+  @Id
+  @Column(name = "data_job_name")
+  private String dataJobName;
+
+  @OneToOne(mappedBy = "dataJobDeployment")
+  @ToString.Exclude
+  @EqualsAndHashCode.Exclude
+  private DataJob dataJob;
+
+  private String deploymentVersionSha;
+
+  private String pythonVersion;
+
+  private String gitCommitSha;
+
+  private Float resourcesCpuRequest;
+
+  private Float resourcesCpuLimit;
+
+  private Integer resourcesMemoryRequest;
```

Comment:
What are the units of measurement for CPU and Memory? Please include them in the name of the variables.

Reply:
@dakodakov For memory will be resourcesMemoryRequestMi. What is your suggestion for the CPU?

Reply:
....CPULimitCores? I don't have a strong opinion.

Reply:
Thanks, I will make the changes as part of the next PR.